packages/theme/src/theme/_tooltip.scss

```diff
@@ -0,0 +1,3 @@
+.tooltip-inner {
+  word-break: break-all;
```

Comment:
Yes, but @charlesnguyen told me the problem with `_` in filename, so OK to merge with `break-all` option. LGTM